scripts/make_module_docs.py

```diff
@@ -36,25 +50,32 @@
 
     if module.extra:
         extra = "\n".join(line.strip() for line in module.extra.split("\n") if line.strip())
-        extra = "\nextra_description: >\n" + indent(extra, "  ")
+        extra += "\n\n"
     else:
         extra = ""
+
     text = f"""\
 ---
 name: {module.name}
 urls: {module.href}
 summary: >
-  {module.info}{extra}
+  {module.info}
 ---
 
-{dedent(docstring)}
```

Comment:
This is where I was thinking that we could have a big HTML comment. Wouldn't make it out of the md > HTML rendering and would help avoid people editing this.

For example, something like:

```html
<!--
~~~~~ DO NOT EDIT ~~~~~
This file is autogenerated from the MultiQC module python docstring.
Do not edit the markdown, it will be overwritten.

File path for the source of this content: {module_file_path}
~~~~~~~~~~~~~~~~~~~~~~~
-->
```

Reply:
Will add!

Reply:
Thanks!